test/setup_transfer.cpp

```diff
@@ -869,7 +869,7 @@ int start_web_server(bool ssl, bool chunked_encoding, bool keepalive)
 	} while (ec);
 
 	char buf[200];
-	snprintf(buf, sizeof(buf), "python ../web_server.py %d %d %d %d"
+	snprintf(buf, sizeof(buf), "python2 ../web_server.py %d %d %d %d"
```

Comment:
> or is the problem that web_server.py is not python3 compatible?

This one, calling `python` refers to `python3` when python3 is installed.

> perhaps the python scripts should all start with "#!/usr/bin/env python"

That won't help on this problem. Nevertheless, that should be added.

>Fixing the python invocations could perhaps be a separate patch though

Then I'll drop it in this PR.